### DISCUSSION TITLE
Budget

### DISCUSSION THREAD

**Discussion by TomAff** [moneymanagerex/moneymanagerex#4179] [Q&A]
Hello all,

I use MMEX since 2012 and now I try to use the budget for the first time.
Is there a good discription or videos to explain it.

For example I have a category for insurances.
Some I have to pay yearly (January), some monthly. 
How can I do this on a monthy budget planning.
Should I set the january amount as yearly and the other month as monthly?

what does "Base on Budget" mean?
Is it just a copy of the numbers?

**Comment by vomikan** [comment]
The budget is an amazing thing. It was only after a long time that I realized what it was and how to use it.
The first thing I would like to note is that initially there was only an annual budget. The monthly budget appeared later and it is independent, does not affect the annual budget in any way. (There is already case #4165 to fix this, but so far so.)
The idea is simple. You specify by category how much you can spend in a year (month) as much as possible. Then you can observe whether these limits are exceeded. The same goes for income.
The easiest way to check this is on private expenses, such as groceries and transportation. It is more difficult with one-time expenses, such as insurance or vacation.

**Comment by TomAff** [comment]
Thanks. 
If I choose "base Budget on" there is a treeview in the navigation.

All income and expenses (yearly, monthly, quarterly) are in my recurring transactions.
I have also stored plan data such as shopping  up there.
So the recurring bookings are actually my budget.

I think I should better create a Report as using the budget tool :)

**Comment by vomikan** [comment]
Yes indeed. Creating a budget from recurring transactions or just transactions would be a great feature. But this has not been implemented yet. If you search, there may even be such an issue.
